File: Tier2/src/main/java/via/sep3/group11/tier2/shared/DTOs/AgreementsByHostDTO.java
package via.sep3.group11.tier2.shared.DTOs;

import via.sep3.group11.tier2.shared.domain.Refugee;

public class AgreementsByHostDTO {
    private String hostEmail;
    private String errorMessage;

    public AgreementsByHostDTO(String hostEmail, String errorMessage) {
        this.hostEmail = hostEmail;
        this.errorMessage = errorMessage;
    }

    public String getHostEmail() {
        return hostEmail;
    }

    public String getErrorMessage() {
        return errorMessage;
    }
}

File: Tier1/Shared/DTOs/AgreementsByHostDTO.cs
namespace Shared.DTOs;

public class AgreementsByHostDTO
{
    public string HostEmail { get; }
    public string ErrorMessage { get; }

    public AgreementsByHostDTO(string hostEmail, string errorMessage)
    {
        HostEmail = hostEmail;
        ErrorMessage = errorMessage;
    }
}